conf/modules/relativelocalizationfilter.xml

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="relativelocalizationfilter" dir="relativelocalizationfilter">
+	<doc>
+		<description>
+			Relative Localization Filter based on Bluetooth Communication between drones
+		</description>
+	</doc>
+	<header>
+		<!-- Main file -->
+		<file name="relativelocalizationfilter.h"/>
+		<!-- Functions files -->
+		<file name="arrayfunctions.h"/>
+		<file name="coordinateconversions.h"/>
```

Comment:
For file naming please use an underscore between natural words, e.g. array_functions.h, coordinate_conversions.h